js/README.md

```diff
@@ -34,12 +34,7 @@ Arrow is a set of technologies that enable big-data systems to process and trans
 
 Apache Arrow is a columnar memory layout specification for encoding vectors and table-like containers of flat and nested data. The Arrow spec aligns columnar data in memory to minimize cache misses and take advantage of the latest SIMD (Single input multiple data) and GPU operations on modern processors.
 
-Apache Arrow is the emerging standard for large in-memory columnar data ([Spark](https://spark.apache.org/), [Pandas](http://wesmckinney.com/blog/pandas-and-apache-arrow/), [Drill](https://drill.apache.org/), ...). By standardizing on a common binary interchange format, big data systems can reduce the costs and friction associated with cross-system communication.
-
-# Related Projects
-
-* [GoAI](http://gpuopenanalytics.com/) -- Arrow-powered GPU analytics
-* [rxjs-mapd](https://github.com/graphistry/rxjs-mapd) -- A MapD Core node-driver that returns query results as Arrow columns
+Apache Arrow is the emerging standard for large in-memory columnar data ([Spark](https://spark.apache.org/), [Pandas](http://wesmckinney.com/blog/pandas-and-apache-arrow/), [Drill](https://drill.apache.org/), [Graphistry](https://www.graphistry.com), ...). By standardizing on a common binary interchange format, big data systems can reduce the costs and friction associated with cross-system communication.
```

Comment:
Can we remove these project links and refer people to the Arrow website to learn more?

Reply:
I recommend keeping a short description here to help ground JS devs. 

Maybe what would be better trimming the technical description of Arrow (+ adding a link here), and using that space to add key details about the JS impl relevant to JS folks. Happy to do so, lmk.

Reply:
Yes, that sounds good. Right below this we can add a "Community" section with a link to the mailing list, JIRA, and Powered By page